src/logic/operate.js

```diff
@@ -22,10 +22,14 @@ const operate = (numberOne, numberTwo, operation) => {
   }
```

Comment:
- [ ] The `%` needs some more work, it should be treated as `percentage`, not `modulus`, e.g. 95 % == .95).

Reply:
solved